pyproject.toml

```diff
@@ -216,12 +216,18 @@ exclude = [
 ]
 #=== End: Ruff
 
+#=== Begin: Flake8
+[tool.flake8]
+max-line-length = 120
+#=== End: Flake8
+
```

Comment:
I suppose this doesn't hurt anything, but it does imply that we are OK with either ruff or flake8.

Reply:
I will remove. Something changed when I updated emacs on my system and it started running both ruff ***and*** flake and the E501 errors were driving me crazy. I think I'll just remove flake8 from my venv.